static/index.html

```diff
@@ -97,6 +119,36 @@ <h3>ordergeschiedenis</h3>
 
         updateStats();
         setInterval(updateStats, 5000);
+
+        const THEME_STORAGE_KEY = "THEME_DARKMODE";
+        const themePickerIcon = document.getElementById('theme-picker-icon');
+
+        function isDarkMode() {
+            return localStorage.getItem(THEME_STORAGE_KEY) === "true";
```

Comment:
Could we default to the system default, using something like `window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches`?

Reply:
Yeah sounds good, will add that now